third_party_integration/huggingface_transformers/BootstrapNAS_HF/0001-Modifications-for-BootstrapNAS-usage.patch

```diff
@@ -852,7 +898,7 @@ index 000000000..87b234560
 +            },
 +            "schedule": {
 +                "list_stage_descriptions": [
-+                    {"train_dims": ["depth", "width"], "epochs": 5, "depth_indicator": 1,  "width_indicator": 3, "init_lr": 5e-5, "epochs_lr": 5}
++                    {"train_dims": ["depth", "width"], "epochs": 5, "depth_indicator": 1,  "width_indicator": 3, "init_lr": 5e-5}
```

Comment:
In HuggingFace lr scheduler, there doesn't seem to be a parameter that matches the functionality of "epochs_lr". (i.e. For HuggingFace lr scheduler, "epochs_lr" should be equal to the value of "epochs"). So I removed "epochs_lr" in the config file.